gtsfm/bundle/bundle_adjustment.py

```diff
@@ -175,6 +180,16 @@ def get_metrics_from_sfm_data(sfm_data: GtsfmData, suffix: str) -> List[GtsfmMet
             )
             ba_metrics.extend(metrics_group=ba_pose_error_metrics)
 
+        if cameras_gt is not None:
+            output_tracks_classification = track_utils.classify_tracks3d_with_gt_cameras(
+                tracks=optimized_data.get_tracks(), cameras_gt=cameras_gt
+            )
+            output_tracks_classification_frequency = Counter(output_tracks_classification)
+
+            for exit_code, count in output_tracks_classification_frequency.items():
+                metric_name = "Filtered tracks triangulated with GT pose: {}".format(exit_code.name)
```

Comment:
Is this just GT pose (extrinsics), or are the GT intrinsics used as well?

Reply:
GT intrinsics are used too. I should correct it